setup.py

```diff
@@ -141,7 +141,7 @@ def get_git_sha() -> str:
         "impala": ["impyla>0.16.2, <0.17"],
         "kylin": ["kylinpy>=2.8.1, <2.9"],
         "mmsql": ["pymssql>=2.1.4, <2.2"],
-        "mysql": ["mysqlclient==1.4.2.post1"],
+        "mysql": ["mysqlclient>=2.1.0"],
```

Comment:
Let's at least restrict the next major:
```suggestion
        "mysql": ["mysqlclient>=2.1.0, <3"],
```